src/Relation/Binary/Reasoning/MultiSetoid.agda

```diff
@@ -35,13 +37,13 @@ begin⟨_⟩_ : ∀ {c l} (S : Setoid c l) {x y} → _IsRelatedTo_ S x y → [ S
 begin⟨_⟩_ S p = EqR.begin_ S p
 
 _≈⟨_⟩_ : ∀ {c l} {S : Setoid c l} x {y z} → [ S ] x ≈ y → _IsRelatedTo_ S y z → _IsRelatedTo_ S x z
-_≈⟨_⟩_ {S = S} = EqR._≈⟨_⟩_ S
```

Comment:
I'd prefer if we just filled in the arguments and defined it in terms of `_≈⟨_⟩_` as before. The less we expose the "hidden" `step` operators the better.

Reply:
I think the `syntax` gets in the way here, also. I can't use the `_≈⟨_⟩_` with a qualified name (`EqR._≈⟨_⟩_`) because it's `syntax` (I think!)

Reply:
I don't think there is any way to rename syntax right now.

What is used in many places is to make the identifier that has syntax attached to it not used for any other purpose, so that at least it becomes possible to selectively import or hide syntax via that identifier.

If a proposal such as in https://github.com/agda/agda/issues/3481#issuecomment-450597651 is implemented, `syntax` may become superfluous.

Reply:
It may be a bit tedious but we can of course hide the name in the import,
redefine it (as an alias to the hidden one) and add a new syntax declaration.
e.g.

```agda
open import Data.Product as P hiding (Σ-syntax)

Σ-syntax = P.Σ-syntax
syntax Σ-syntax A (λ x → B) = Σ[ x ∶ A ] B
```